resteasy-core/src/main/java/org/jboss/resteasy/plugins/server/servlet/HttpServletResponseWrapper.java

```diff
@@ -154,9 +227,11 @@ public boolean suppressExceptionDuringChunkedTransfer() {
       private AsyncOperation lastAsyncOperation;
```

Comment:
I think this is not assigned at all.

Reply:
And we don't need this block - https://github.com/resteasy/resteasy/blob/9331416b193f326cfb96832f39fb564bbcf963c2/resteasy-core/src/main/java/org/jboss/resteasy/plugins/server/servlet/HttpServletResponseWrapper.java#L348-L351